tests/robot-tests/tests/libs/common.robot

```diff
@@ -1,5 +1,5 @@
 *** Settings ***
-Library     SeleniumLibrary    timeout=${timeout}    implicit_wait=${implicit_wait}    run_on_failure=do this on failure
+Library     SeleniumLibrary    timeout=${timeout}    implicit_wait=%{IMPLICIT_WAIT}    run_on_failure=do this on failure
```

Comment:
I'd also change `timeout=${timeout}` to use `timeout=%{TIMEOUT}`

That said, I'm not sure this change is necessary. It's possible at the point these variables are used in this line, the `*** Variables ***` section will already be run, so `${timeout}` and `%{TIMEOUT}` will be the same. But to be safe I'd use the environment variable on this line i.e. `timeout=%{TIMEOUT}`

Reply:
Good catch, I've changed this now to %{TIMEOUT}